website/content/docs/connect/cluster-peering/index.mdx

```diff
@@ -1,32 +1,58 @@
 ---
 layout: docs
-page_title: Service Mesh - What is Cluster Peering?
+page_title: Cluster Peering Overview
 description: >-
-  Cluster peering establishes communication between independent clusters in Consul, allowing services to interact across datacenters. Learn about the cluster peering process, differences with WAN federation for multi-datacenter deployments, and technical constraints.  
+  Cluster peering establishes communication between independent clusters in Consul, allowing services to interact across datacenters. Learn how cluster peering works, its differences with WAN federation for multi-datacenter deployments, and how to troubleshoot common issues.  
 ---
 
-# What is Cluster Peering?
+# Cluster peering overview
 
-You can create peering connections between two or more independent clusters so that services deployed to different partitions or datacenters can communicate.
+This topic provides an overview of cluster peering, which is a process that connects two or more independent clusters so that services deployed to different partitions or datacenters can communicate.
```

Comment:
```suggestion
This topic provides an overview of cluster peering, which lets you connect two or more independent Consul clusters so that services deployed to different partitions or datacenters can communicate.
```